pkg/reconciler/ingress/resources/secret_test.go

```diff
@@ -239,6 +247,13 @@ func TestMakeWildcardSecrets(t *testing.T) {
 	}}
 	for _, c := range cases {
 		t.Run(c.name, func(t *testing.T) {
+			if c.originSecret.Labels == nil {
+				c.originSecret.Labels = map[string]string{}
+			}
+			c.originSecret.Labels[network.SecretInformerLabelKey] = c.originSecret.Name
+			for _, s := range c.expected {
+				s.Labels[network.SecretInformerLabelKey] = c.originSecret.Name
+			}
```

Comment:
just pass an arbitrary label pair to the original secret. Copy should get the same pair.